### PR TITLE
Fix query preventing automation cards from rendering

### PR DESCRIPTION
**Pull request by myrdstom**
#### How should this be manually tested?
1. On terminal, run command ...

#### Any background context you want to provide?
[#167915721](https://www.pivotaltracker.com/story/show/167915721)

#### Screenshots (If applicable)

### CODE REVIEW

File: server/controllers/automationController.js [hunk 1]
```diff
@@ -43,16 +43,10 @@ const order = [
...
-  const automationIds = await models.sequelize.query(automationRawQuery, { ...querySettings }, options);
-  const allData = await automation.findAll({
+  return await automation.findAll({
     ...options,
-    where: {
-      id: {
-        $in: automationIds.map($da => $da.id),
-      },
-    },
+
```
Comment (johnkegz) [inline server/controllers/automationController.js]: @myrdstom you have removed the functionality for pagination and if you are to check your frontend, the pagination for the number of automations returned for a particular page no longer exists.

File: server/controllers/automationController.js [hunk 2]
```diff
@@ -43,16 +43,10 @@ const order = [
  * @return  {array}                       array of automation objects
  */
 async function getAutomationDataFromIds(automationRawQuery, querySettings = {}, options = {}) {
-  const automationIds = await models.sequelize.query(automationRawQuery, { ...querySettings }, options);
-  const allData = await automation.findAll({
+  return await automation.findAll({
```
Comment (johnkegz) [inline server/controllers/automationController.js]: @myrdstom you have removed the functionality for pagination and if you are to check your frontend, the pagination for the number of automations returned for a particular page no longer exists.

Comment (09arnold) [inline server/controllers/automationController.js]: This just does a general findAll without making use of the automationIDs, which results in all automations being returned all the time when this function is called

File: server/controllers/automationController.js [hunk 3]
```diff
@@ -43,16 +43,10 @@ const order = [
  * @return  {array}                       array of automation objects
  */
 async function getAutomationDataFromIds(automationRawQuery, querySettings = {}, options = {}) {
-  const automationIds = await models.sequelize.query(automationRawQuery, { ...querySettings }, options);
```
Comment (09arnold) [inline server/controllers/automationController.js]: What is the reason for removing this line and the `where` clause in the lines below?

File: server/controllers/automationController.js (hunk 2 above)
Comment (myrdstom) [reply server/controllers/automationController.js]: Noted.. will work on it and revert

Comment (myrdstom) [reply server/controllers/automationController.js]: Ahh... thanks, Noted, lemme look into it

File: server/controllers/automationController.js (hunk 3 above)
Comment (myrdstom) [reply server/controllers/automationController.js]: I noticed the effect in retrospect and I will rectify this

Comment (09arnold) [reply server/controllers/automationController.js]: This line is still missing resulting in `automationIds is not defined` error

Comment (myrdstom) [reply server/controllers/automationController.js]: `const automationIds = await models.sequelize.query(automationRawQuery, { ...querySettings }, options);`
The line is there and everything seems to be working on my end. did you pull the most recent changes?

**Pull request merged**